### PR TITLE
Feature/native portals

### PR DESCRIPTION
**Pull request by GavanWilhite**

### CODE REVIEW

File: src/components/NPortal.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import NativeComponent from './NativeComponent';
+
+/**
+*/
+class NPortal extends NativeComponent {
+	constructor(){ super('n-portal'); }
+	get schema(){
+		return {
+			targetSpace: {type: 'string'},
+			targetEntity: {type: 'selector'}
```
Comment (GavanWilhite) [inline src/components/NPortal.js]: targetEl?

I think that's slightly more consistent as you are selecting an element that represents an entity.

File: src/components/NativeComponent.js [hunk 2]
```diff
@@ -46,10 +46,7 @@ class NativeComponent extends AFrameComponent
 		safeDeepSet(mesh.userData, ['altspace', 'collider', 'enabled'], false);
 		altspace.addNativeComponent(mesh, this.name);
 
-		if (this.sendUpdates) {
-			//to pass defaults
-			altspace.updateNativeComponent(mesh, this.name, this.data);
-		}
+		this.update();
```
Comment (GavanWilhite) [inline src/components/NativeComponent.js]: no if check?

File: src/components/NPortal.js (hunk 1 above)
Comment (brianpeiris) [inline src/components/NPortal.js]: A-Frame docs refer to `<a-entity>` elements as entities. Also wouldn't want to mislead people into thinking they could select any old HTML element.

File: src/components/NativeComponent.js (hunk 2 above)
Comment (brianpeiris) [inline src/components/NativeComponent.js]: There's an if check in `update()`. We kinda don't use `sendUpdates` anyway.

**Pull request merged**